manifests/repomd_gpg.pp

```diff
@@ -0,0 +1,64 @@
+# == Class: certs
+# Sets up the GPG key for Pulp repository metadata signing
+class certs::repomd_gpg (
+  $enable                  = $::certs::enable_repomd_gpg,
+  $regenerate              = $::certs::regenerate_repomd_gpg,
+  $dir                     = $::certs::repomd_gpg_dir,
+  $user                    = $::certs::repomd_gpg_user,
+  $group                   = $::certs::repomd_gpg_group,
+  $gpg_name                = $::certs::repomd_gpg_name,
+  $comment                 = $::certs::repomd_gpg_comment,
+  $email                   = $::certs::repomd_gpg_email,
+) {
+  if $enable {
+
+    $gpg_key_params = "Key-Type: RSA\nKey-Length:4096\nExpire-Date: 0\n"
```

Comment:
Could you make this a parameter too? Experience has taught me that at some point someone will want to override this.